ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/HiveMetaStoreAuthorizer.java

```diff
@@ -213,15 +214,46 @@ public final List<String> filterCatalogs(List<String> catalogs) throws MetaExcep
 
   @Override
   @Deprecated
-  public List<TableMeta> filterTableMetas(String catName, String dbName,List<TableMeta> tableMetas)
+  public List<TableMeta> filterTableMetas(String catName, String dbName, List<TableMeta> tableMetas)
       throws MetaException {
-    return filterTableMetas(tableMetas);
+    if (LOG.isDebugEnabled()) {
```

Comment:
No need to check LOG.isDebugEnabled(), since LOG.debug will only output, when debug level logging is enabled.